src/biogeochem/CNCIsoFluxMod.F90

```diff
@@ -85,7 +84,7 @@ subroutine CIsoFlux1(num_soilc, filter_soilc, num_soilp, filter_soilp,         &
     character(len=*)                      , intent(in)    :: isotope         ! 'c13' or 'c14'
     !
```

Comment:
I think I did something very similar in the fates bgc call sequence PR, here is the commit for your reference: https://github.com/ESCOMP/CTSM/commit/0676ac2ae11ea910cab5cdae075d4c119493ef26

Reply:
REsolving.